### PR TITLE
MLDBFB-558 need to cast ts issue demo

### PR DESCRIPTION
**Pull request by FinchPowers**

### CODE REVIEW

**Comment by FinchPowers** [comment]
Désolé pour le diff, mon vim semble avoir fait le ménage.

**Comment by nicolaskruchten** [comment]
Oh I had already created MLDB-1706 ticket with test cases in the issue... Also, I don't think I agree that `horizontal_avg` should work, or that strings should auto-cast to timestamps :)

**Comment by FinchPowers** [comment]
Ok misunderstanding. I'll rewrite the test.

**Comment by FinchPowers** [comment]
Updated.

**Comment by nicolaskruchten** [comment]
can you add the TIMESTAMP keyword before the timestamps plz? Otherwise the tests are the same: string comparisons. We don't auto-cast ISO strings as timestamps

File: testing/MLDB-461_horizontal_ops_test.py [hunk 1]
```diff
@@ -208,5 +207,29 @@ def test_horizontal_latest(self):
...
+        self.assertEqual(res[1][1], 'b')
 
+        res = mldb.query("""
+            SELECT horizontal_min({'2015-01-01T00:00:00Z',
+                                   '2016-01-01T00:00:00Z'})
+        """)
+        self.assertEqual(res[1][1], '2015-01-01T00:00:00Z')
+
+        res = mldb.query("""
+            SELECT horizontal_max({CAST ('2015-01-01T00:00:00Z' AS TIMESTAMP),
```
Comment (nicolaskruchten) [inline testing/MLDB-461_horizontal_ops_test.py]: can we just use `TIMESTAMP '2015-01-01T00:00:00Z'` which is the preferred way of creating timestamps?

File: testing/MLDB-461_horizontal_ops_test.py [hunk 2]
```diff
@@ -208,5 +207,29 @@ def test_horizontal_latest(self):
...
+        """)
+        self.assertEqual(res[1][1], 'a')
+
+        res = mldb.query("""
+            SELECT horizontal_max({'a', 'b'})
+        """)
+        self.assertEqual(res[1][1], 'b')
 
+        res = mldb.query("""
+            SELECT horizontal_min({'2015-01-01T00:00:00Z',
```
Comment (nicolaskruchten) [inline testing/MLDB-461_horizontal_ops_test.py]: these should be timestamps

**Comment by FinchPowers** [comment]
Comme `SELECT TIMESTAMP '2015-01-01T00:00:00Z'` ?

**Comment by mailletf** [comment]
@FinchPowers oui ajouter TIMESTAMP avant chaque string ayant un timestamp, de manière à le caster, et tester le horizontal_min sur un timestamp et non une string. C'était ça ta question?

**Pull request merged**